entrypoint.sh

```diff
@@ -228,6 +229,7 @@ start_system() {
   seed_db
   echo "Starting Cachet! ..."
   php artisan config:cache
+  envsubst '\$PORT' < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
```

Comment:
I think this may need to actually happen along with all of the `sed` replacement such as https://github.com/CachetHQ/Docker/blob/master/entrypoint.sh#L186 -- the CI failures seem to be related to a permissions error when trying to write to `/etc/nginx/conf.d/default.conf` as the non-root user.